src/ini.cr

```diff
@@ -19,4 +19,27 @@ class INI
     end
     ini
   end
+
+  # Generates an INI-style configuration from a given hash.
+  #
+  # ```
+  # INI.build({"foo" => {"a" => "1"}}, " ") # => "[foo]\na = 1\n\n"
+  # ```
+  def self.build(ini, space : String = "") : String
+    String.build do |str|
+      build str, ini, space
+    end
+  end
+
+  # Appends INI data to the given IO.
+  #
+  def self.build(io : IO, ini, space : String = "")
```

Comment:
Personally I think that having the space be configurable as any string is a bit too much and makes it possible to create invalid INI files. Why not make it a boolean which switches between `""` and `" "`. Regardless, I think 1 space should be the default, not zero. We can always change this default later.